src/runtime/pkg/device/drivers/vfio.go

```diff
@@ -192,12 +211,13 @@ func (device *VFIODevice) Load(ds config.DeviceState) {
 	device.GenericDevice.Load(ds)
 
 	for _, dev := range ds.VFIODevs {
-		device.VfioDevs = append(device.VfioDevs, &config.VFIODev{
+		var vfioDev config.VFIODev = config.VFIOPCIDev{
```

Comment:
That's a bit confusing as coded, since it has field that appears to accept all kinds of VFIO device, but in reality only creates VFIO PCI devices and ignores the type.

Reply:
The code change has been finalized to cover PCI and AP cases.

Reply:
Yes, I saw that later. Sorry.